tendrl/commons/objects/node_context/__init__.py

```diff
@@ -93,6 +93,11 @@ def render(self):
     def save(self, update=True, ttl=None):
         super(NodeContext, self).save(update)
         status = self.value + "/status"
+        if self.status == "UP" and ttl == None:
```

Comment:
`ttl is None` ?

Reply:
This check will always be assigning TTL value when node status is up, if we are saving node object with status "up" without TTL then TTL is an optional parameter in a save function, so it will check status is up and TTL None then it will assigning TTL while saving an object.

 TTL is important for the watcher function because we are marking node status is down based on TTL only. I saw sometimes node status is updated as up without TTL. So it is affecting watcher to modify status as down when a node is actually down.

We can always remember to give TTL wherever we use node object save. so i put that in a common place.

Reply:
All I meant was instead of saying `ttl == None` we should say `ttl is None` :)

Reply:
oh :)